generators/app/templates/src/utils/navUtils.js

```diff
@@ -15,3 +18,14 @@ export const getPreviousRouteName = navigationState => {
   }
   return navigationState.routes[navigationState.index - 1].routeName;
 };
+
+export function inferRoute(screenObj) {
+  const screenName = Object.keys(screenObj)[0];
+  return {
+    [Routes[screenName]]: {
```

Comment:
maybe we could extract `Routes[screenName]` as a `const` since it's used several times. Could help with readability too!

Reply:
We could make something like this if you are agree:

```
export function inferRoute(screenObj) {
  const screenName = Routes[Object.keys(screenObj)[0]];
  return {
    [screenName]: {
      screen: screenObj[screenName],
      navigationOptions: screensNavOptions[screenName],
      ...screenObj
    }
  };
}
```

Reply:
are you sure that you wouldn't have an issue with `screen: screenObj[screenName]` by doing this replacement you suggest?

Reply:
I think you'd also have to add some kind of check in case `Object.keys(screenObj)[0]` doesn't exist so the application doesn't crash... maybe an optional chaining there? @guidoprinc

Reply:
yeah, maybe

Reply:
> are you sure that you wouldn't have an issue with `screen: screenObj[screenName]` by doing this replacement you suggest?

I'm not sure, we should try.